consensus/src/runway/mod.rs

```diff
@@ -629,6 +671,24 @@ where
         self.send_consensus_notification(NotificationIn::NewUnits(units_to_move))
     }
 
+    fn status_report(&self) {
+        let UnitStoreStatus {
+            forkers,
+            size,
+            height,
+            top_row,
+        } = self.store.get_status();
+        let runway_status: RunwayStatus<H> = RunwayStatus {
```

Comment:
It might be nicer to create this struct from a `UnitStoreStatus` and the two remaining refs. You could then implement `Display` for `UnitStoreStatus` and use it in `RunwayStatus`'s implementation. Perhaps you could then also use constructors properly, `pub` fields are sorta icky. <_<

Reply:
removed more `pub`s